src/Storages/System/attachInformationSchemaTables.cpp

```diff
@@ -35,8 +35,9 @@ static constexpr std::string_view schemata = R"(
         `DEFAULT_CHARACTER_SET_SCHEMA` Nullable(String),
         `DEFAULT_CHARACTER_SET_NAME` Nullable(String),
         `SQL_PATH` Nullable(String)
-    ) AS
-    SELECT
+    )
+    SQL SECURITY INVOKER
```

Comment:
Why "INVOKER"?

Reply:
Because apparently, we don't want to give all other users access to all tables
![image](https://github.com/ClickHouse/ClickHouse/assets/19871338/8d479039-0096-4b0d-8eb0-f0c51e8625de)

Reply:
But `ATTACH VIEW` without `SQL SECURITY` always means `ATTACH VIEW SQL SECURITY INVOKER`, right?